unit-tests/live/CMakeLists.txt

```diff
@@ -19,3 +19,122 @@ dl_file( ${PP_Rosbag_Recordings_URL} recordings [pointcloud]_all_combinations_de
 
 # for 3D/projection-from-recording.bag
 dl_file( ${PP_Rosbag_Recordings_URL} recordings single_depth_color_640x480.bag OFF )
+
+# for post-processing/post-processing-filters.py
+# Define a function to download files
+function(dl_file url subdir filename)
```

Comment:
Where did this download took place in the legacy code?

Reply:
It is defined in unit-tests\live-test.cmake, it was downloaded to temp dir. Here I changed it to recording dir like the other bag files downloaded in this cmake.